internal/rest/kubernetes.go

```diff
@@ -171,3 +171,135 @@ spec:
       securityContext:
         runAsUser: 0
 `
+
+const k8sDeploymentChart = `apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: observiq-cluster-collector
+  name: observiq-cluster-collector
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: observiq-cluster-collector
+  labels:
+    app: observiq-cluster-collector
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  - namespaces/status
+  - nodes
+  - nodes/spec
+  - nodes/stats
+  - nodes/proxy
+  - pods
+  - pods/status
+  - replicationcontrollers
+  - replicationcontrollers/status
+  - resourcequotas
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
+  - deployments
+  - replicasets
+  - statefulsets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - extensions
+  resources:
+  - daemonsets
+  - deployments
+  - replicasets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  - cronjobs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+    - autoscaling
+  resources:
+    - horizontalpodautoscalers
+  verbs:
+    - get
+    - list
+    - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: observiq-cluster-collector
+  labels:
+    app: observiq-cluster-collector
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: observiq-cluster-collector
+subjects:
+- kind: ServiceAccount
+  name: observiq-cluster-collector
+  namespace: default
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: observiq-cluster-collector
+  labels:
+    app: observiq-cluster-collector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: observiq-cluster-collector
+  template:
+    metadata:
+      labels:
+        app: observiq-cluster-collector
+    spec:
+      serviceAccount: observiq-cluster-collector
+      containers:
+        - name: otc-container
+          image: observiq/observiq-otel-collector:{{ .version }}
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              memory: 100Mi
+              cpu: 50m
+            limits:
+              memory: 100Mi
```

Comment:
I think we can bump these up. The low values are suitable of quick lightweight testing.

```suggestion
            requests:
              memory: 200Mi
              cpu: 100m
            limits:
              memory: 200Mi
```

I am seeing the following when using an aggressive collection interval (5s), constant events, and a handful of processors.
```
NAME                                          CPU(cores)   MEMORY(bytes)   
observiq-cluster-collector-77d689d47d-w8hrj   17m          69Mi    
```